test/worlds/elevator.sdf

```diff
@@ -0,0 +1,1333 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <world name="elevator">
+
+    <physics name="1ms" type="ignored">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+    </physics>
+    <plugin
+      filename="ignition-gazebo-physics-system"
+      name="ignition::gazebo::systems::Physics">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-sensors-system"
+      name="ignition::gazebo::systems::Sensors">
+      <render_engine>ogre2</render_engine>
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-scene-broadcaster-system"
+      name="ignition::gazebo::systems::SceneBroadcaster">
+    </plugin>
```

Comment:
I think `Sensors`, `SceneBroadcaster`, the light and maybe even the ground plane aren't needed for the test

Reply:
Done. On a side note, the test passes, but now, in citadel, I get this error

```
[Dbg] [SimulationRunner.cc:422] Creating PostUpdate worker threads: 2
[Dbg] [SimulationRunner.cc:435] Creating postupdate worker thread (0)
[Err] [JointPositionController.cc:214] [JointPositionController]: Detected an invalid <joint_index> parameter. The index specified is [0] but the joint only has [0] index[es]. This controller will be ignored
```